internal/manager/controllerdef.go

```diff
@@ -254,7 +254,7 @@ func setupControllers(
 			// knative is a special case because it existed before we added feature gates functionality
 			// for this controller (only) the existing --enable-controller-knativeingress flag overrides
 			// any feature gate configuration. See FEATURE_GATES.md for more information.
-			Enabled: featureGates[gatewayFeature] || c.KnativeIngressEnabled,
```

Comment:
I wonder what was this about 🤔 Why knative was connected with gateway feature gate 🤔 ?

Reply:
I'm glad you pointed this out, because in trying to fix it I made a typo. It should be the `knativeFeature` that's checked, not `gatewayFeature`.

Reply:
Soooooooooooooo basically this was a bug causing users that use knative ingresses and **not** enabling gateway feature gate to not be able to use their knative ingresses 😲 ? 

Maybe we should add that to changelog as well?

Reply:
The thing was, nobody was using it. But yeah I suppose you're right all the same. added to changelog.